src/docker-compose

```diff
@@ -154,7 +157,7 @@ comp_start() {
         cd "$manifests_dir"
         $DOCKER_COMPOSE_CMD \
             --project-name "$project_name" \
-            up --detach > ../compose.log 2>&1
+            up --detach --wait --wait-timeout $WAIT_TIMEOUT > ../compose.log 2>&1
```

Comment:
Same here, I'm not sure if these two are supported by the `docker-compose` implementation.